vita3k/config/include/config/config.h

```diff
@@ -57,5 +57,5 @@ struct Config {
     bool color_surface_debug = false;
     bool hardware_flip = false;
     bool performance_overlay = false;
-    std::string backend = {};
+    int backend = -1;
```

Comment:
Why -1 ?

Reply:
So command line options won't overwrite the config file options if nothing is specified.

Reply:
set 0 works

Reply:
Are you sure you're getting the Vulkan gui and not the OpenGL gui that looks exactly the same? The only difference afaik is the messages in log that give you the GL version, and maybe some graphical differences.

Reply:
and personnaly, after very workis with kd, you can keep std:string wtih write OpenGL, i have already done works gui for works like that

Reply:
Leave it as string for user to easier to config. Leave it all lowercase

Reply:
yes ^^ anjd i prepare pr after for implement select backend render

Reply:
From my experience it needs an empty value or else po is going to overwrite it. -1 is the empty value for now. Config file can't have -1 so its safe but po can. @Zangetsu38 I'm pretty sure you're just getting the OpenGL gui.

I'll make it lowercase.

Reply:
How about std::tolower so the user can enter any combination they please?

Reply:
i get vulkan and opengl with switch in my setting dialog :)

Reply:
😍😍😍😍😍😍😍😍😍😍😍😍😍😍